test/unit/iam-token-manager.test.js

```diff
@@ -119,294 +141,5 @@ describe('iam_token_manager_v1', () => {
     const token = await instance.getToken();
     expect(token).toBe(ACCESS_TOKEN);
     expect(requestMock).not.toHaveBeenCalled();
-    done();
-  });
-
```

Comment:
All of these tests below were consolidated and are covered by the tests for the new IAM request based token manager